### PR TITLE
docs(kafka-log) add support for api_version config

### PR DESCRIPTION
**Pull request by murillopaula**

### CODE REVIEW

File: app/_hub/kong-inc/kafka-log/index.md [hunk 1]
```diff
@@ -91,7 +91,11 @@ params:
       default: "`50000`"
       description: |
         Maximum number of messages that can be buffered in memory in asynchronous mode.
-
+    - name: api_version
+      required: false
+      default: "`0`"
+      description: |
+        API version of a Produce. Allowed values: 0, 1 and 2.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/kafka-log/index.md:98]:
```suggestion
        API version of a Produce request. Allowed values: `0`, `1`, or `2`.
```

**Comment by murillopaula** [comment]
@lena-larionova @GayleNeumann can we merge this? this is available in EE 1.5

**Comment by lena-larionova** [comment]
>  this is available in EE 1.5

Echoing comment I made in your other PR: does this need to be only in 1.5.x, and not in 1.3-x?

**Comment by murillopaula** [comment]
> Echoing comment I made in your other PR: does this need to be only in 1.5.x, and not in 1.3-x?

yeah this is not available in EE 1.3... but this plugin doesn't have a `1.3-x` neither a `1.5-x` thing, should I create it?

**Comment by lena-larionova** [comment]
Yeah, you'll need to create a new file for version 1.5.x, call it `index`, and rename the old one as `1.3-x`. Then, add a 1.5.x entry to this file: https://github.com/Kong/docs.konghq.com/blob/master/app/_data/extensions/kong-inc/kafka-log/versions.yml

**Comment by murillopaula** [comment]
appreciate the reviews @lena-larionova what do you think about this PR now?

**Comment by lena-larionova** [comment]
@murillopaula sorry for the delay, I lost track of this one. Looks good now - I just updated the version formatting.

**Pull request merged**